google-forwarding.md

```diff
@@ -0,0 +1,20 @@
+# Email Forwarding for Gmail Users
+For the best experience, we reccomend that users login directly to their PTA email boxes using the Gmail app on their phone, or in the web browser.  This allows a user to send and recieve email directly from their PTA account.  
+
+However, some users may prefer not to have a separate PTA email address.  If this is the case, they can forward their PTA email to a personal address.  Their PTA email box will still have a copy of all messages recieved.
+
+Follow this guide to forward all PTA email to your personal address.
```

Comment:
This line is a little redundant with the header of the setup section below. I suggest a merge into the header.